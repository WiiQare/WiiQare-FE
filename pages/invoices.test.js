import "@testing-library/jest-dom";
import Invoices from "./invoices";
import { render, screen } from "@testing-library/react";

describe("Invoices page", () => {
  it("should render the invoices page", () => {
    const { container } = render(<Invoices />);
    expect(container).toMatchSnapshot();
  });
});
