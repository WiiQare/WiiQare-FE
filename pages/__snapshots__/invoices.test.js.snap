// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Invoices page should render the invoices page 1`] = `
<div>
  <div>
    <h2
      class="text-black font-w700 mt-4"
    >
      Invoices
    </h2>
  </div>
</div>
`;
